packages/ra-ui-materialui/src/button/UpdateButton.stories.tsx

```diff
@@ -44,7 +45,7 @@ const getDataProvider = () =>
                     id: 1,
                     title: 'Lorem Ipsum',
                     body: 'Lorem ipsum dolor sit amet',
-                    views: 1000,
+                    views: 500,
```

Comment:
why change this line?

Reply:
Cause it is easier to find the string '500' than find '1 000' with space on tests